session-10/task/goroutines/task1.go

```diff
@@ -0,0 +1,21 @@
+package goroutines
+
+import (
+	"fmt"
+	"time"
+)
+
+func printNumbers() {
+	for i := range 5 {
+		fmt.Println(i + 1)
+		time.Sleep(time.Millisecond)
+	}
+
+}
+
+func RunTask1() {
+	fmt.Println("Main function started")
+	go printNumbers()
+	fmt.Println("Main function ended")
+	time.Sleep(time.Second)
```

Comment:
_Note: Since Goroutines run concurrently, output order may vary slightly._
NIT: Bu Note-u nezere alaraq 500ms-den biraz artiq zaman alacagini estimate ederek, time.Sleep()-i function end-den sonra qoyardim. 
```suggestion
	fmt.Println("Main function started")
	go printNumbers()
	time.Sleep(time.Second)
	fmt.Println("Main function ended")
```